### PR TITLE
Added alt text to images on style guide.

### PR DESCRIPTION
**Pull request by balagan73**
#425 

- Added alt text to images on the style guide.
 
image

- Fix for WCAG issues of social share icons were already ported.

0.5/4h

### CODE REVIEW

**Comment by balagan73** [comment]
@amitaibu This is waiting for your review. The wcag problems this issue was originally created were already fixed, but I added some missing alt test on the style guide.

File: web/modules/custom/server_style_guide/src/Controller/StyleGuideController.php [hunk 1]
```diff
@@ -324,7 +325,7 @@ protected function getMediaVideo(): array {
    *   Render array.
    */
   protected function getCardsWithImageForNews(): array {
-    $image = $this->buildImage($this->getPlaceholderImage(300, 200));
+    $image = $this->buildImage($this->getPlaceholderImage(300, 200), 'Card1 image');
```
Comment (amitaibu) [inline web/modules/custom/server_style_guide/src/Controller/StyleGuideController.php:328]: Let's change

```diff
-  protected function buildImage(string $url, string $alt = '') {
+  protected function buildImage(string $url) {
```

and have some dummy `alt placeholder` added inside `buildImage`

**Comment by balagan73** [comment]
@amitaibu Travis is green now.

**Review by amitaibu — APPROVED** [review]
Thanks

**Pull request merged**